src/js/themes/base.js

```diff
@@ -1646,6 +1646,11 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       panel: {
         // extend: undefined,
       },
+      step: {
+        small: 1,
```

Comment:
Why do we only step by 1 on a small screen?

Reply:
If you are on a phone and the tabs move by a step of 3, with longer tab names you could potentially miss seeing one of the tabs. So for small screens where fewer tabs can fit, it felt better to only move by one tab at a time